packages/react-mui-shared/src/components/JsonViewer/index.tsx

```diff
@@ -1,13 +1,8 @@
-import { FC, lazy } from "react";
+import { FC } from "react";
 import { Box } from "@mui/material";
 import { useTheme } from "@mui/material/styles";
 import { useReactJsonViewHighlight, Highlight } from "./utils";
-import { ReactJsonViewProps } from "react-json-view";
-
-let ReactJson: FC<ReactJsonViewProps> = () => null;
-if (typeof window !== "undefined") {
-  ReactJson = require("react-json-view").default;
```

Comment:
@cj-f Do you remember why this `require()` is/was needed?

Reply:
Somewhat, I don't remember the exact error but essentially the package could only be used within the browser and trying to use the package within nextjs was causing issues, which I believe I got around by dynamically loading the package although that was a bad end-user experience, which where I then came across something mentioning to do this

Reply:
I just threw this as-is into our pangea-integration-nextjs-authn example and it renders fine, so I think we're good to go?

Reply:
Ok, lets try it